application/modules/admin/views/scripts/evaluate/edit-tb-evaluation.phtml

```diff
@@ -423,10 +423,8 @@ if (
                                                                                                                                                                                                                                                                                                 } ?> />
                         </td>
                         <td style="vertical-align: top;text-align:center;width: 13%;">
-                            <select id="rifResistance<?php echo $counter; ?>" name="rifResistance[<?php echo $index; ?>]" class="isRequired form-control input-sm" title="Please enter the Rif Resistance for this sample" placeholder="Please enter the Rif Resistance for this sample" <?php if (!$rifResistanceEnabled) {
-                                                                                                                                                                                                                                                                                                echo "readonly";
-                                                                                                                                                                                                                                                                                            } ?>>
-                                <option value="">-- Select --</option>
+                            <select id="rifResistance<?php echo $counter; ?>" name="rifResistance[<?php echo $index; ?>]" class="isRequired form-control input-sm" title="Please enter the Rif Resistance for this sample" placeholder="Please enter the Rif Resistance for this sample" <?php if (!$rifResistanceEnabled) {                                                                                                                                                                                                                                                                              } ?>>

```

Comment:
Please explain why you made this change?

Reply:
I have not made any changes here.